routes/dms.js

```diff
@@ -262,7 +278,7 @@ module.exports = function () {
   })
 
   // MUST come last in order to catch all the publisher pages
-  router.get('/:owner', async (req, res, next) => {
+  router.get('/:owner', param(['owner']).escape(), validationCheck, async (req, res, next) => {
```

Comment:
@sagargg why do we need it here but not for other routes?

Reply:
@anuveyatsu mentioned [issue](https://github.com/datopian/client-national-grid-shared/issues/48) matched to this specific routes so, implemented only for this. Let me know if you think we should apply on other routes as well.

Reply:
@sagargg yes, that was just an example but we should implement this for all routes.

Reply:
okay, I will add on all routes.

Reply:
@anuveyatsu  I used both `createDOMPurify` and `express validator` for params XSS sanitization.